src/backend/_scss/application.scss

```diff
@@ -348,11 +419,17 @@ a {
 }
 
 .sponsor-container {
-  background: $white;
+  background: $white-classic-background;
 }
 
 .sponsor-row {
-  margin: 0;
+  font-weight: bold;
+  color: black;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Color `black` should be written in hexadecimal form as `#000000`](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)